### ISSUE TITLE
Ground truth of case 151673

### ISSUE THREAD

**Issue by JunMa11** [jianhuupenn/SpaGCN#15]
Dear @jianhuupenn ,

Thanks for sharing the nice code.

Could you please share the ground truth and code to compute the adjusted rand index in Figure 3-b?

Best regards,
Jun

**Comment by jianhuupenn** [comment]
The SpaGCN predictions calculated with p=0.5, random seed, torch seed, NumPy seed all seting to 100, using Environment 1 listed on the main page.
The ARIs are calculated using "sklearn.metrics.adjusted_rand_score()"
The ground truth is obtained from the original study's website.

**Comment by JunMa11** [comment]
Hi @jianhuupenn , 

Thanks for your quick reply.
I do not find the ground truth on http://research.libd.org/spatialLIBD/
![1640029955](https://user-images.githubusercontent.com/19947331/146824584-95d7a29c-6684-4e47-b913-74d92272782f.png)

Could you please share the link to the ground truth?
